beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -328,3 +326,15 @@ func (s *Service) hasBlockAndState(ctx context.Context, blockRoot [32]byte) bool
 	hasState := hasStateSummary || s.cfg.beaconDB.HasState(ctx, blockRoot)
 	return hasState && s.cfg.chain.HasBlock(ctx, blockRoot)
 }
+
+func seenAttCacheKey(slot primitives.Slot, committeeID primitives.CommitteeIndex, aggregationBits []byte) string {
+	totalLen := 8 + 8 + len(aggregationBits)
+	key := make([]byte, totalLen)
+	binary.LittleEndian.PutUint64(key[:8], uint64(slot))
+	binary.LittleEndian.PutUint64(key[8:16], uint64(committeeID))
+	copy(key[16:], aggregationBits)
+
+	// Avoid copying to reduce allocation when casting. It is guaranteed to be immutable as the
+	// slice is only created in this method and isn't accessed/returned anywhere else.
+	return unsafe.String(&key[0], len(key)) // #nosec G103
```

Comment:
Can you use the same bytesutil.UnsafeCastToString from #14591 ?

Reply:
Will do after that PR is merged in and remove this